paraatm/tests/aviationr_example.py

```diff
@@ -0,0 +1,44 @@
+import os
+import numpy as np
+import pandas as pd
+from paraatm.simulation_method.aviationr import AviationRisk
+import matplotlib.pyplot as plt
+cur_dir = os.path.dirname(os.path.abspath(__file__))
+data_dir = os.path.join(cur_dir, '..', 'sample_data/')
+
+cfg = {'fp_file': data_dir + 'aviationR/data/TRX_DEMO_SFO_PHX_GateToGate.trx',  # flight plan file
+       'mfl_file': data_dir + 'aviationR/data/TRX_DEMO_SFO_PHX_mfl.trx',  # mfl file
+       'data_file': data_dir + 'aviationR/data/', # required data for accident simulation and risk estimation
+       'model_file': data_dir + 'aviationR/model/', # Pre-trained model
+       'sim_time': 8000}  # total simulation time
+
+# call
+sim = AviationRisk(cfg)
+
+result  = sim.simulation()  # call simulation function using NatsSimulationWrapper
+if result is not None:
+    track = pd.read_table(data_dir + 'aviationR/data/trajectory.csv', sep=',', skiprows=[0,1,2,3,4,6,7,8,9], index_col=0)
+
+    fig, ax = plt.subplots(3, 1,figsize=(15,30))
```

Comment:
This is the biggest matplotlib plot that I have ever seen!  Please reduce the figure size so that it can fit onto the screen of typical monitors (I'm using 1980x1080 resoltuion and the figure does not fit within the screen).  I suggest something like:
```python
    fig, ax = plt.subplots(1, 3,figsize=(12,8))
```
Note this also uses columns instead of rows, which seems to make better use of the available. space.  You probably also want to either remove or shorten the x-axis tick labels in the first plot, as they take up a ton of space and there's really no good way to format the plot to allow for them.

Another tip is to use `fig.subplots_adjust` to fine tune the figure settings.  Generate an interactive version of your plot and click on the subplots adjustment tool to fine tune things like space at the top/bottom and space between the columns.  Then you can include these settings in your script via `fig.subplots_adjust`.  You don't have to pass a value for every option, just the ones that you change from their defaults.